linalg/petscinternals.hpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#ifndef MFEM_PETSCINTERNALS
+#define MFEM_PETSCINTERNALS
+
+// Error handling
```

Comment:
without the proper include statements from petsc and MFEM headers this file is useless. Try compiling

```
#include "petscinternals.hpp"

int main()
{
int err = 0;
MPI_Comm comm;
CCHKERRQ(comm,err);
PetscObject obj;
PCHKERRQ(obj,err);
return 0;
}
```

Reply:
Good catch, fixed.